tools/lou_checkyaml.c

```diff
@@ -808,8 +808,9 @@ main(int argc, char *argv[]) {
 	// FIXME: problem with this is that
 	// LOUIS_TABLEPATH=$(top_srcdir)/tables,... does not work anymore because
 	// $(top_srcdir) == .. (not an absolute path)
-	if (chdir(dir_name))
-		error(EXIT_FAILURE, EIO, "Cannot change directory to %s", dir_name);
+	if (i > 0)
```

Comment:
Can't you put this into one if statement?
``` C
if ((i>0) && chdir(dir_name))
  error(EXIT_FAILURE, EIO, "Cannot change directory to %s", dir_name);
```

Reply:
yes, kinda definitely :-)

Reply:
Hmm I'm not so sure actually. It's a matter of taste really. I compare it with using `&&` vs. `if-then` in bash. There is something to say for both styles. One is shorter but the other is clearer. You may change it if it really bothers you.

Reply:
OK, fine with me